README.md

```diff
@@ -340,16 +491,55 @@ After writing to the captureOuput.pcap file you can read the file with the follo
 
```

Comment:
What's up with this "Expand all" above? Can it be written as a regular text? Because it looks empty in between the two commands and the button to show those 2 lines in between is not very visible.